matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/algorithms/megolm/MXMegolmEncryption.kt

```diff
@@ -304,6 +301,16 @@ internal class MXMegolmEncryption(
         } else {
             Timber.i("## CRYPTO | shareUserDevicesKey() : no need to sharekey")
         }
+
+        if (noOlmToNotify.isNotEmpty()) {
+            //XXX offload?, as they won't read the message anyhow?
+            notifyKeyWithHeld(
```

Comment:
This is now sent after the code with `if (haveTargets)`, I think it is OK, but could you confirm please?

Reply:
yes that's what i wanted
(i even wonder if we could offload to other thread to speed up sharing to the one that will be able to read the message)

Reply:
ok, thanks